### DISCUSSION TITLE
How i can get user-agents and logs in database?

### DISCUSSION THREAD

**Discussion by masihgh** [bilfeldt/laravel-route-statistics#20] [Ideas]
hi i need useragents for statics how can i do for thats problem?

**Comment by bilfeldt** [comment]
@masihgh Great question. I suggest:

1. Add an extra column to the database table
2. Create a new model extending the one from the package, but override the `getLogAttributes()` function (can call the parent and simply add an extra value)
3. Configure Themis package to use the new model
